kleva-j/iReporter
# Feature/#162503997/delete redflag using a database

#### What does this PR do?
- create functions to get delete a record of an incident from a database

#### Description of the task to be completed
 - create an extended function of the database protocol to delete a red-flag record
 - create functions to handle requests to delete a red-flag record

#### How should this be manually tested?
- Clone the repository
- Check out into the branch Feature/#162503997/delete-a-redflag-using-a-database
- Install all dependencies
- run `npm start` to run the app successfully.

#### What are the relevant pivotal tracker stories?
[162503997](https://www.pivotaltracker.com/story/show/162503997)
